newrelic-python.ini

```diff
@@ -61,32 +66,11 @@ security.mode = RASP
 security.validator_service_endpoint_url = ws://your_validator_system_ip:54321
 security.resource_service_endpoint_url = http://your_validator_system_ip:54322
 
-# log level
-security.log_level = DEBUG
-
-# security file settings
-security.sec_home_path = /tmp
-
 # vulnerabilty detection disable
 security.detection.disable_rci = false
 security.detection.disable_rxss = false
 security.detection.disable_deserialization = false
 
```

Comment:
Upper 3 flags should be -
security.detection.rci.enabled = true
security.detection.rxss.enabled = true
security.detection.deserialization.enabled = true

Reply:
Updated

Reply:
mark them as true instead of false